operators/pkg/instctrl/containers_test.go

```diff
@@ -144,6 +155,7 @@ var _ = Describe("Generation of the container based instances", func() {
 			ContainerEnvOpts: containerOpts,
 		}
 
+		ctx, _ = clctx.TenantInto(ctx, &tenant)
```

Comment:
If the previous two comments are meaningful, this is not necessary.

Reply:
Ok, removed